### ISSUE TITLE
InputFormField default props appends -invalid classname

### ISSUE THREAD

**Issue by brandongregoryscott** [AndcultureCode/AndcultureCode.JavaScript.React.Components#82]
Pulled in this component for a small personal project and I noticed it was setting the `-invalid` classname by default. Looks like it is controlled by the `isValid` prop, but it feels like maybe that shouldn't be something controlled from props - the presence of `errorMessage` or `errorMesssages` props might make more sense. At the very least, "invalid" should not be the default state.

**Comment by TheDataDesigner** [comment]
Shouldn't we be styling invalid inputs using the `:invalid` pseudo-class? There shouldn't be any cases where we are making an input look invalid when it actually isn't, so we should try to leverage the built-in selectors for this kind of stuff.

**Comment by brandongregoryscott** [comment]
Yeah, that makes sense @alexgetty - either way, the current functionality is a bit misleading
